sdk/tests/client/node_api/core.rs

```diff
@@ -226,7 +229,7 @@ async fn test_get_included_block_metadata() {
     let metadata_response = client.get_included_block_metadata(&transaction_id).await.unwrap();
 
     assert_eq!(metadata_response.block_id, block_id);
-    assert!(!metadata_response.parents.is_empty());
+    assert_eq!(metadata_response.tx_state.unwrap(), TransactionState::Finalized);
```

Comment:
```suggestion
    assert_eq!(metadata_response.tx_state, Some(TransactionState::Finalized));
```

Just makes it more clear what's wrong if we somehow had an empty state.